### PR TITLE
TAD Landing page copy changes

### PR DESCRIPTION
**Pull request by akenzua**
[TP11777](https://maps.tpondemand.com/RestUI/Board.aspx#page=board/5682182231901749200&appConfig=eyJhY2lkIjoiNzlGODk3MENDQkM0N0FCM0ZFQzkyNEVFODEyREM0M0EifQ==&boardPopup=userstory/11777/silent)

This PR added copy changes to the landing page and styled the tooltips.

### CODE REVIEW

**Comment by davidtrussler** [comment]
The copy changes look good on the landing page but it looks like there are still copy changes to be made on the listing page. 

I'm also a little concerned a the lack of Welsh copy throughout, or is there a plan to add this when all the English copy is complete. If that is the case can we ensure there is a TP card for this so that it doesn't get left out? 🙏

**Comment by akenzua** [comment]
> The copy changes look good on the landing page but it looks like there are still copy changes to be made on the listing page.
> 
> I'm also a little concerned a the lack of Welsh copy throughout, or is there a plan to add this when all the English copy is complete. If that is the case can we ensure there is a TP card for this so that it doesn't get left out? 🙏

The plan is to have the correct version in English first, then translate it to welsh in one TP card. Is there any copy change you noticed i have missed on this particular TP card?

**Comment by davidtrussler** [comment]
> The plan is to have the correct version in English first, then translate it to welsh in one TP card. Is there any copy change you noticed i have missed on this particular TP card?

OK, good to hear there's a plan in place for the Welsh copy 👍 

On the missing copy:
- it looks like the update is missing for the `Filter by type of insurance` tooltip
- maybe also the `Length of trip` filter - though I'm not sure what the requirement is here: should there be a general tooltip for the filter rather than for `select one of the durations`?

**Comment by akenzua** [comment]
> > The plan is to have the correct version in English first, then translate it to welsh in one TP card. Is there any copy change you noticed i have missed on this particular TP card?
> 
> OK, good to hear there's a plan in place for the Welsh copy 👍
> 
> On the missing copy:
> 
> * it looks like the update is missing for the `Filter by type of insurance` tooltip
> * maybe also the `Length of trip` filter - though I'm not sure what the requirement is here: should there be a general tooltip for the filter rather than for `select one of the durations`?

Thanks, I have corrected that of insurance type, the other one is correct, they both have different tooltips.

**Review by davidtrussler — APPROVED** [review]
LGTM! 👍

File: components/listingsPage/filters.js [hunk 1]
```diff
@@ -223,7 +223,11 @@ const Filters = ({ t }) => {
             <TooltipText
               minWidth="200px"
               side="left"
-              text={t("toolTips.age")}
+              text={
+                <Paragraph color="#515151" textSize="14px">
+                  {t("toolTips.age")}
+                </Paragraph>
+              }
```
Comment (andreferreiradlw) [inline components/listingsPage/filters.js:230]: Can you remove the margin for each of the `Paragraphs` used inside the Tooltips? The padding on the tooltip doesn't look uniform like this. Maybe worth creating a custom `styled(Paragraph)` element and also apply the text size there and reuse it?

**Review by andreferreiradlw — APPROVED** [review]

**Pull request merged**